klvdata/misb0601.py

```diff
@@ -59,6 +59,11 @@ class Checksum(BytesElementParser):
     Initialized from bytes value as BytesValue.
     """
     key = b'\x01'
+    TAG = 1

```

Comment:
If possible, can we keep the scope of this PR down to adding just the additional element parsers and not additional functionality? I look forward to the day where klvdata can handle these features but I think it would be great if this PR focused on adding the missing parsers for which there is already support for (i.e. local set, using the existing Element Parsers and ST 0102 Nest Local Set).

Reply:
as you want. In this way the parsers are complete, I think